scripts/process_notebooks.py

```diff
@@ -139,71 +139,49 @@ def should_process(path):
         # Clean the original notebook and save it to disk
         print(f"Writing complete notebook to {nb_path}")
         with open(nb_path, "w") as f:
-            nb_clean = clean_notebook(nb)
+            if nb_path.startswith("tutorials"):
```

Comment:
Rather than writing a pattern like

```python
if boolean_expression:
    var = True
else:
    var = False
```

it is cleaner to write

```python
var = boolean_expression
```

e.g. 

```
clear_outputs = nb_path.startswith("tutorials")
```

Reply:
If you rename the parameter name as suggested below, you don't even really need to define the variable, e.g.

```python
nb_clean = clean_notebook(nb, clear_outputs=nb_path.startswith("tutorials"))
```

Though doing it in two steps is totally fine if you feel it's easier to read.

Reply:
The one-step syntax is very clear. I have changed to this one.